package.json

```diff
@@ -102,7 +105,7 @@
     "api-watch": "nodemon -e \".ts\" --exec npm run babel-node-api",
     "babel-node-api": "babel-node -r dotenv/config -x \".ts\" ./src/api/api.ts",
     "build-api": "npx tsc --build ./tsconfig-api.json",
-    "test-api": "npx jest --testTimeout=5000 --config=./src/api/jest.config.ts",
+    "test-api": "npx jest --testTimeout=5000 --config=./src/api/jest.config.ts --runInBand",
```

Comment:
MSW is not happy running unless using `--runInBand`. Runs tests serially instead of worker pool of child processes.